Mesh_3/include/CGAL/Mesh_3/features_detection/features_detection.h

```diff
@@ -1547,10 +1543,10 @@ return {
   create_polyline(0.,1., P(1./2,2./3,0.),P(1./2,1./2,1.),
                  [](double z) { return P(-(-7*z*z + 7*z + ((7*z*z - 8*z + 2)/(2*(9*z*z - 10*z + 3)) + std::sqrt(13*z*z*z*z - 36*z*z*z + 40*z*z - 20*z + 4)/(2*(9*z*z - 10*z + 3)))*(9*z*z - 10*z + 3) - 2)/(z*(3*z - 1)),(7*z*z - 8*z + 2)/(2*(9*z*z - 10*z + 3)) + std::sqrt(13*z*z*z*z - 36*z*z*z + 40*z*z - 20*z + 4)/(2*(9*z*z - 10*z + 3)),z); },
                  prec),
-  create_polyline(1./2,(8.06225-7.)/2, P(1./2,0.,2./3),P((8.06225-7.)/2,1./3,(3.+8.06225)/14),
+  create_polyline(1./2,(std::sqrt(65)-7.)/2, P(1./2,0.,2./3),P((std::sqrt(65)-7.)/2,1./3,(3.+std::sqrt(65))/14),
```

Comment:
In a second commit the `std::sqrt(2)` is replaced with `CGAL_SQRT2`, similar for some other but why isn't this done for `std::sqrt(65)`?

Reply:
`std::sqrt(2)`, `std::sqrt(3)`, and `std::sqrt(5)` are used A LOT in this file. `std::sqrt(65)` has only a few occurrences so we decided it is not worth creating a macro for it

Reply:
indeed the `std::sqrt(2)`, `std::sqrt(3)`, and `std::sqrt(5)` are used quite a few times compared to `std::sqrt(65)` but from a consistency point of view I think it would be better to also have a CGAL_SQRT65.